File: android/SmartIRHub/app/src/main/java/com/ms8/smartirhub/android/_tests/dev_playground/remote_layout/asymmetricgridview/AGVRecyclerViewAdapter.java
package com.ms8.smartirhub.android._tests.dev_playground.remote_layout.asymmetricgridview;


import androidx.recyclerview.widget.RecyclerView;

public abstract class AGVRecyclerViewAdapter<VH extends RecyclerView.ViewHolder>
    extends RecyclerView.Adapter<VH> {
  public abstract AsymmetricItem getItem(int position);
}

File: android/SmartIRHub/app/src/main/java/com/ms8/smartirhub/android/remote_control/views/asymmetric_gridview/AGVRecyclerViewAdapter.kt
package com.ms8.smartirhub.android._tests.dev_playground.remote_layout.asymmetricgridview_k

import androidx.recyclerview.widget.RecyclerView

abstract class AGVRecyclerViewAdapter<VH : RecyclerView.ViewHolder> : RecyclerView.Adapter<VH>() {
    abstract fun getItem(position: Int): AsymmetricItem
}